test_fake.py

```diff
@@ -0,0 +1,42 @@
+import pendulum
```

Comment:
Is there a reason for adding a new module? why not use datetime.datetime.now?

Reply:
yes. You are right, we should use datetime.